client/my-sites/site-settings/delete-site/index.jsx

```diff
@@ -186,10 +186,14 @@ module.exports = React.createClass( {
 	},
 
 	_deleteSite: function() {
-		var site = this.state.site;
-		SiteListActions.deleteSite( site );
-		window.scrollTo( 0, 0 );
-		page( '/stats' );
+		this.setState( { showDialog: false } );
+
+		SiteListActions.deleteSite( this.state.site, function( success ) {
+			if ( success ) {
+				window.scrollTo( 0, 0 );
```

Comment:
wouldn't this scrollTo make more sense in the stats controller? I mean, each time the stats router triggers the `overview` method there, it probably make sense to move the scroll to the top